### ISSUE TITLE
Github About text needs to be updated

### ISSUE THREAD

**Issue by jqnatividad** [ckan/ckan#5672]
**CKAN version**
N/A

**Describe the bug**
europeandataportal.eu no longer runs on CKAN.

**Expected behavior**
Examples best comes in threes :)  - add another high-profile site instead - ideally, from the private sector using an enterprise use case, not an "open data" use case.

**Comment by amercader** [comment]
The EU Data Portal [technical documentation](https://www.europeandataportal.eu/en/about/documentation) still lists CKAN as the backend powering the Drupal frontend. Do you know if this setup has changed more recently?

**Comment by jqnatividad** [comment]
What spurred opening the issue was when I looked at their gitlab, CKAN was listed in the "Deprecated" section.

https://gitlab.com/european-data-portal
https://gitlab.com/european-data-portal/deprecated-components

As luck would have it, they just released this report this week.  

https://www.europeandataportal.eu/en/highlights/sustainability-open-data-portals-infrastructures-reports-pt-4

I found it curious that even though they identified building the portal the most expensive part, there was no mention of CKAN, open source, or helping reducing build cost by funding pillar technologies of their infrastructure, like CKAN using one of their proposed financial models.

https://www.europeandataportal.eu/en/highlights/sustainability-open-data-portals-infrastructures-reports-pt-4
